tests/cases/conformance/jsdoc/jsdocImportTypeReferenceToClassAlias.ts

```diff
@@ -0,0 +1,15 @@
+// @noEmit: true
+// @allowJs: true
+// @checkJs: true
+// @Filename: mod1.js
+class C {
+    s() { }
+}
+module.exports.C = C
```

Comment:
This is needed because this line isn't treated like an alias declaration, right?

Reply:
Yes, and the fixup code in getTypeFromJSDocValueReference didn't handle import types before this.